src/constants/idos.ts

```diff
@@ -0,0 +1,88 @@
+interface AnnouncedIDOItem {
+  id: number
+  title: string
+  description?: string
+  status: string
+  launchpad: string
+  announcementUrl: string
+  projectIconLocation: string
+  projectUrl: string
+  startTime?: string
+  endTime?: string
+}
+
+export const IDO_LIST: Array<AnnouncedIDOItem> = [
+  {
+    id: 5,
+    title: 'Kalao',
+    description: '',
+    status: 'Upcoming',
```

Comment:
ex. the value here should be IDO_STATUS_UPCOMING which would be an export defined above

Reply:
changed IDO items to use the IDO_STATUS constants